content/issues/2022-10-27-slo1-error-budget-drop-postmortem.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "3/4 of SLO1 Error Budget consumed in 6 weeks"
+date: "2022-09-01T01:00:00+02:00"
+resolved: true
+resolvedWhen: "2022-10-12T01:00:00+02:00"
+section: issue
+severity: notice
+---
+
+(Mostly for our internal use since the source data are not publicly available.)
+
+During the 6 weeks since the beginning of September the [SLO1](https://github.com/packit/research/blob/main/error-budgets/SLOs-SLIs.md#slo1-changes-to-github-prs-receive-a-status-update-within-15-seconds-in-99-of-cases)
```

Comment:
```suggestion
During the 6 weeks since the beginning of September the [SLO1](https://packit.dev/docs/service-level-objectives/#slo1-changes-to-github-prs-receive-a-status-update-within-15-seconds-in-99-of-cases)
```
:)